ci/dev/helm.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I was going to add this as a `yarn` script but then it'd require the CI job to install Node just to have `yarn` so I opted for a separate bash script

Reply:
Would it make sense to put `helm kubeval ci/helm-chart` directly in the yaml?

Reply:
Good idea! I doubt either of us have run this locally and then we have one less script to maintain.